layouts/partials/meta.html

```diff
@@ -2,7 +2,7 @@
 <meta name="HandheldFriendly" content="True">
 <meta name="MobileOptimized" content="320">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<meta name="referrer" content="no-referrer">
+<meta name="referrer" content="unsafe-url">
```

Comment:
Is this related to untappd?

Reply:
No, not required for untappd.

Reply:
Sorry, I realize now that this was resolved by #109 anyways.